.dockerignore

```diff
@@ -1,4 +1,87 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# vim swap files
+*swp
+*swo
+*~
+
+# Ping data files
+*.ping
+*.pings
+*.population*
+*.percent
+*.cities
+populations
+
+# local config files
+#*.json
+
+# Discarded code snippets
+build.sh
+*-fast.yaml
+detritus/
+
+# Dotnet Core ignores
+*.swp
+*.*~
+project.lock.json
+.DS_Store
+*.pyc
+nupkg/
+
+# Visual Studio Code
+.vscode
+
+# User-specific files
+*.suo
+*.user
+*.userosscache
+*.sln.docstates
+
+# Build results
+[Dd]ebug/
+[Dd]ebugPublic/
+[Rr]elease/
+[Rr]eleases/
+x64/
+x86/
+build/
+bld/
+[Bb]in/
+[Oo]bj/
+[Oo]ut/
+msbuild.log
+msbuild.err
+msbuild.wrn
+
+# Visual Studio 2015
+.vs/
+
+# Goland
+.idea/
+
+# Nodejs files placed when building Hugo, ok to allow if we actually start using Nodejs.
+package.json
```

Comment:
I have not used Hugo before, just wondering if the developers are still able to install the required npm modules without this package.json file

Reply:
I create an isolated install of nodejs just for Hugo. Installing the node_modules/ does some funny things we'd like to avoid bleeding elsewhere. If we start requiring nodejs we can have a checked in node_modules/ directory and just use that but for now it doesn't make sense to leak buidl context.